guides/micronaut-jms-aws-sqs/groovy/src/main/groovy/example/micronaut/DemoController.groovy

```diff
@@ -0,0 +1,21 @@
+package example.micronaut
+
+import groovy.transform.CompileStatic
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+
+@CompileStatic
+@Controller  // <1>
+class DemoController {
+
+    private final DemoProducer demoProducer
+
+    DemoController(DemoProducer demoProducer) {  // <2>
+        this.demoProducer = demoProducer
+    }
+
+    @Get("/demo")  // <3>
```

Comment:
The controller which has a side effect it publishes to a Queue. It should be mapped to POST instead of Get. 

I would annotate the method with `@Status(NO_CONENT)` and respond a 204 instead 200.